x-pack/plugins/security_solution/server/lib/siem_migrations/rules/task/rule_migrations_task_client.ts

```diff
@@ -28,7 +28,7 @@ import type {
 } from './types';
 import { ActionsClientChat } from './util/actions_client_chat';
 
-const ITERATION_BATCH_SIZE = 50 as const;
+const ITERATION_BATCH_SIZE = 15 as const;
```

Comment:
How should we find the connector type from here? Do I need to pass it down so we can set it only for bedrock or keep it like this for now?

Reply:
I think we can improve the batching strategy, for all connector types, the current approach is pretty basic. We could process rules progressively instead of starting all the rules in the batch at the same time. Also, we need to implement the exponential backoff on top of it.

Let's keep it with the 15 for now, and start thinking about improving the batching strategy in general.